src/expressions.rst

```diff
@@ -1281,8 +1281,20 @@ The :t:`evaluation` of an :t:`addition expression` proceeds as follows:
 #. :dp:`fls_jjmc1xgny77`
    The :t:`right operand` is evaluated.
 
+#. :dp:`fls_NcLf4o1dpniS`
+   If the types of both operands are the same :t:`integer type` or
```

Comment:
I would change the beginning of the sentence to

If the :t:`type` of both :t:`[operand]s` is the same ...